cpp/tests/large_strings/parquet_tests.cpp

```diff
@@ -69,3 +70,57 @@ TEST_F(ParquetStringsTest, ReadLargeStrings)
   // go back to normal threshold
   unsetenv("LIBCUDF_LARGE_STRINGS_THRESHOLD");
 }
+
+TEST_F(ParquetStringsTest, ChunkedReadLargeStrings)
+{
+  // Construct a table with one large strings column > 2GB
+  auto const wide = this->wide_column();
+  auto input      = cudf::concatenate(std::vector<cudf::column_view>(120000, wide));  ///< 230MB
+
+  int constexpr multiplier = 12;
+  std::vector<cudf::column_view> input_cols(multiplier, input->view());
+  auto col0 = cudf::concatenate(input_cols);  ///< 2.70GB
+
+  // Expected table
+  auto const expected    = cudf::table_view{{col0->view()}};
+  auto expected_metadata = cudf::io::table_input_metadata{expected};
+  expected_metadata.column_metadata[0].set_encoding(
+    cudf::io::column_encoding::DELTA_LENGTH_BYTE_ARRAY);
+
+  // Write to Parquet
+  auto const filepath = g_temp_env->get_temp_filepath("ChunkedReadLargeStrings.parquet");
+  cudf::io::parquet_writer_options out_opts =
+    cudf::io::parquet_writer_options::builder(cudf::io::sink_info{filepath}, expected)
+      .compression(cudf::io::compression_type::ZSTD)
+      .stats_level(cudf::io::STATISTICS_NONE)
+      .metadata(expected_metadata);
+  cudf::io::write_parquet(out_opts);
+
+  // Read with chunked_parquet_reader
+  size_t constexpr pass_read_limit =
+    size_t{8} * 1024 * 1024 *
```

Comment:
Chosen by trial and error so the chunking would result in two subpasses. First subpass reading ~2.24GB strings and second one reading the rest of 338MB.

Reply:
This appears to be a very large file?
How long does this test take to run? Also, how long under compute-sanitizer?

Reply:
It’s the same column from another large string gtest (`CaseTest.Lower`). Takes roughly 2s to run and 5.5s with `compute-sanitizer —rmm_mode=cuda`

Reply:
Ok. Thanks for checking!

Reply:
Without compute sanitizer, the peak GPU mem usage seen in nvidia-smi is roughly 3.4GB and with compute sanitizer, it’s roughly 14.3GB

Reply:
@vuule for vis

Reply:
I'm okay with two seconds. I get triggered closer to 10s mark :D